cmake_config.hpp.in

```diff
@@ -0,0 +1,2 @@
+#define VERSION_MAJOR @vcf-validator_VERSION_MAJOR@
```

Comment:
Can this file be placed inside the `inc` folder or wouldn't it work that way?

Reply:
done in next commit

Reply:
@cyenyxe, did you mean only the cmake_config.hpp.in? or the cmake_config.hpp as well?

Reply:
I am fine with tracking only the .in in the repository as the other auto-generated, but I think both files should be placed inside `inc` for consistency.

Reply:
fixed in next commit